project/src/main/ui/analog-event-translator.gd

```diff
@@ -23,13 +23,8 @@ var _down_pressed := false
 var _left_pressed := false
 var _right_pressed := false
 
-## Joypad action mappings populated by scanning the InputMap.
```

Comment:
This change in functionality should result in the header comment on Line 1 changing as well.

Reply:
Changed comment to `sends out JoypadButton events...`